lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -219,6 +230,7 @@ class TestGetHtmlMethod(BaseTestXmodule):
     '''
     Make sure that `get_html` works correctly.
     '''
+    maxDiff = None
```

Comment:
Is this intentional ?

Reply:
yes. https://docs.python.org/2/library/unittest.html#unittest.TestCase.maxDiff